File: src/main/java/rbq/wtf/lycoris/client/wrapper/wrappers/annotation/repeat/WrapEnums.java
package rbq.wtf.lycoris.client.wrapper.wrappers.annotation.repeat;

import rbq.wtf.lycoris.client.wrapper.wrappers.annotation.WrapEnum;

import java.lang.annotation.*;

@Documented
@Target(ElementType.FIELD)
@Retention(RetentionPolicy.RUNTIME)
/**
 * Auto setup field
 */
public @interface WrapEnums {
    WrapEnum[] value();
}

File: src/main/java/rbq/wtf/lycoris/client/wrapper/wrappers/annotation/repeat/WrapEnums.kt
package rbq.wtf.lycoris.client.wrapper.wrappers.annotation.repeat

import rbq.wtf.lycoris.client.wrapper.wrappers.annotation.WrapEnum

@MustBeDocumented
@Target(AnnotationTarget.FIELD)
@Retention(AnnotationRetention.RUNTIME)
/**
 * Auto setup field
 */
annotation class WrapEnums(vararg val value: WrapEnum)